linchpin/linchpin.conf

```diff
@@ -88,3 +89,23 @@ level = logging.INFO
 
 [ansible]
 console = True
+
+[playbook_pre_states]
+up = preup
+destroy = predestroy
+
+[playbook_post_states]
+up = postup
+destroy = postdestroy
+
+[states]
+# in future each state will carry a comma seperated substates
+preup = 'preup'  
+predestroy = 'predestroy'
+postup = 'postup'
+postdestroy = 'postdestroy'
+
+[extns]
+resource = '.output'
+inventory = '.inventory'
```

Comment:
These single-quotes are included in the value, so the inventory file is created with them included, e.g. "filename'.inventory'"

```python
>>> from configparser import ConfigParser
>>> cp = ConfigParser()
>>> cp.read_string("""
... [section]
... opt = 'value'
... """)
>>> cp.get('section', 'opt')
"'value'"
```
^ note the included single quotes